pkg/runctl/config_test.go

```diff
@@ -68,10 +96,48 @@ func Test_loadPipelineRunsConfig_CompleteConfigMap(t *testing.T) {
 	assert.NilError(t, err)
 	expectedConfig := &pipelineRunsConfigStruct{
 		NetworkPolicy: "networkPolicy1",
+		JenkinsfileRunnerPodSecurityContextRunAsUser:  int64Ptr(1111),
+		JenkinsfileRunnerPodSecurityContextRunAsGroup: int64Ptr(2222),
+		JenkinsfileRunnerPodSecurityContextFSGroup:    int64Ptr(3333),
 	}
 	assert.DeepEqual(t, expectedConfig, resultConfig)
 }
 
+func Test_loadPipelineRunsConfig_InvalidValues(t *testing.T) {
+	for i, p := range []struct {
+		key, val string
+	}{
+		{pipelineRunsConfigKeyPSCRunAsUser, "a"},
```

Comment:
Also check with -1, 0, 65536

Reply:
As discussed offline, those values would be parsed successfully. We do not want to implement validation of these integer values, as this would be redundant to what Tekton/Kubernetes will do.